### DISCUSSION TITLE
Allow custom field for KANBAN board columns.

### DISCUSSION THREAD

**Discussion by timharris777** [community/community#5933] [Projects and Issues] [answered]
Currently the KANBAN board view in the new projects only allows for columns to be the "Status" field. I want to be able to view all my issues on the KANBAN board organized into columns based on custom fields values.

**Comment by douglaslassance** [comment]
At least for any existing single select field.

**Comment by erinmez** [comment]
I would like to see this too! We would like to have an issue in different Kanban Views (Development, Enablement, Releasing) with different columns. In our case it would be convenient to have 3 custom fields with different single selection values that we can group by for the different Kanban Views.

**Comment by wagoodman** [comment]
Same --I've got a column called "urgency" that helps illustrate how soon we should tackle things on a roadmap. Having a board view organized into ordered columns would help get a sense of "how many things" are "how far out". I'm doing this with the row-oriented view, which is great, but just wanted to put in another use case here.

**Comment by matthewmayer** [comment]
Also for assignee. I'd like to be able to see at a glance how many issues each developer has assigned.

**Comment by lslv1243** [comment]
That is what I thought would happen when activating the group by for a custom single select field in the board view, but to my surprise it was only being grouped by status. 
I believe it is a duplicate of: https://github.com/github/feedback/discussions/5571
But good news, it is on their roadmap: https://github.com/github/roadmap/issues/282

**Reply by matthewmayer** [reply]
I agree that would be the obvious way to make it work in the UI

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - we just shipped this! https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/ 

Check it out in the board view menu and let me know what you think!

**Reply by erinmez** [reply]
Hi @azenMatt , great news, I already looked into it, it works great! 

Just a small thing though, it would be great if one could remove/hide by default the "No xxx" column (the one which contains the tickets that have not been assigned a value in a specific kanban view).

Reason being, I would like to have a custom single select field for each Team and use one kanban view per Team. A product manager prepares tickets by picking the single select value for the team that needs to handle it. Thus, the other teams don't need to see all unprepared tickets.

**Reply by azenMatt** [reply]
That makes sense - early next year I think we will have an addition ready so you can customise which columns you can see just like you can with the table.

**Reply by kob123** [reply]
Hey @azenMatt I've just started using custom board view - nice! Been patiently waiting for this feature.

Fyi though I noticed the `Status` isn't displaying in the cards for some reason. 

This would be replaceable in the [example board from the gif on the release notes](https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/) by changing to Priority, but if you still want to see Status on each card. 

Under Configuration and Visible fields, clicking `Status` on and off again doesn't change anything in the cards display. 

Same with for `Repository`. 

`Milestones` & `Labels` and other custom added fields **do** seem to work / turn on/off just fine though.

Not sure if relevant but ftr I have modified the names of the Status options from their defaults to`ASAP ⏰ ` & `High ✴️ ` etc.

Cheers!

**Reply by azenMatt** [reply]
Great point @kob123 - this was an oversight - we will get that fixed 

Respository we decided to always keep shown on the cards as it felt closer to a structural part of the card than the optional fields (but open to feedback on that!)

**Reply by kob123** [reply]
Awesome thanks @azenMatt.

You know, I didn't actually even see/notice the repo in at the very top of the card there, and was only trying to see where it showed up when I selected/deselected it and didn't notice any change so thought it wasn't there! I see it now and do like it there.

So yeah if it's always there I would remove the option to selected/deselected it in Configuration and Visible fields.

Thanks!

**Comment by matthewmayer** [comment]
It's still not possible to use Assignee as a column?

**Reply by azenMatt** [reply]
not yet!

**Comment by polevaultweb** [comment]
repo as a column would be so good too
